media-video/ffmpeg/ffmpeg-4.0.1.recipe

```diff
@@ -55,25 +56,29 @@ PROVIDES="
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libbz2$secondaryArchSuffix
-	lib:libgnutls$secondaryArchSuffix
 	lib:libmodplug$secondaryArchSuffix
 	lib:libogg$secondaryArchSuffix
 	lib:libspeex$secondaryArchSuffix
 	lib:libtheoradec$secondaryArchSuffix
 	lib:libtheoraenc$secondaryArchSuffix
 	lib:libvorbis$secondaryArchSuffix
 	lib:libvorbisenc$secondaryArchSuffix
-	lib:libvpx$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
 	"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	REQUIRES+="
+		lib:libvpx$secondaryArchSuffix
+		lib:libgnutls$secondaryArchSuffix
+	"
```

Comment:
I guess you meant:
```diff
 	REQUIRES+="
 		lib:libvpx$secondaryArchSuffix
 		lib:libgnutls$secondaryArchSuffix
-	"
+		"
```
Hmm, instead, you could, maybe, remove one tab in lines 69, 70 and 71, and keep line 72 unchanged.

(Same remarks for the closing double quotes (at line 165) of the `BUILD_REQUIRES` bloc.)

BTW, I haven't tested your hack yet, but it seems it is an excellent idea.

Regarding the way this PR will be merged, I think "**Rebase** and merge" is the right thing to do, and I guess you will be using this method, (instead of "Squash and merge").

I would, however amend the very first commit of this PR:
* first, revert the `sed s/ffmpeg/ffmpeg4/` @extrowerk did in `PROVIDES{,_avdevice,_devel,_tools}`;
* second, remove `ffmpeg-4.0.recipe` and `ffmpeg-4.0.patchset`.

This will allow to have slightly smaller `git diff` for both the first and second commits of this PR.
And the "**Rebase** and merge" method will be perfect for this PR.